plugins/module_utils/network/vyos/rm_templates/ntp_global.py

```diff
@@ -50,11 +50,11 @@ def __init__(self, lines=None, module=None):
             "name": "allow_clients_delete",
             "getval": re.compile(
                 r"""
-                ^set\ssystem\sntp\sallow-clients
+                ^set\sservice\sntp\sallow-clients
```

Comment:
It looks like this needs the `(?P<path>system|service)?` to match both the old and new versions

Reply:
Also, in this RegEx, do you need the `?` after the capture group?  This makes the capture group optional and thus would match on “set ntp allow-clients” as well as “set system allow clients”, which wouldn’t be likely to exist but would be incorrect if used in `parsed` mode.